lib/linux_admin/hosts.rb

```diff
@@ -36,6 +38,20 @@ def update_entry(address, hostname, comment = nil)
       end
     end
 
+    def hostname=(name)
```

Comment:
If you are setting ip address, would you ensure that if you set a different ip address, it clears out the old ip address from `/etc/hosts`
If you are changing the hostname, do we also clear that out.

man, the use cases around this are complicated.